### ISSUE TITLE
Cannot view the markup view for firebugs

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#1555]
**Description:**

I think there might be a recent regression, which is causing issues seeing the elements panel for firebugs and other sites

**Steps to Reproduce:**

1. go to https://replay.io/view?id=45042611-16bb-4c21-9963-82e1e8f05ccc
2. click "Elements"

<img width="1919" alt="Screen Shot 2021-01-22 at 6 26 24 AM" src="https://user-images.githubusercontent.com/254562/105502947-db4db300-5c7a-11eb-8b03-28c40bed92c1.png">

**Comment by jasonLaster** [comment]
I'm seeing an exception here `at Promise.all._pause.sendMessage.then._rules (node.ts:219)`

**Comment by jasonLaster** [comment]
Also noticed that this request did not have a response

```
{"id":23,"sessionId":"f8f4f537-98e6-47cf-987d-9bc37aa5b1e8/32fd14c9-bb0a-4c93-8a7d-ab495c7a8ee1","pauseId":"41","method":"DOM.querySelector","params":{"node":"1","selector":"body"}}
```

**Comment by jasonLaster** [comment]
This request didn't have a response either

```
{"id":17,"sessionId":"f8f4f537-98e6-47cf-987d-9bc37aa5b1e8/32fd14c9-bb0a-4c93-8a7d-ab495c7a8ee1","pauseId":"41","method":"DOM.getDocument","params":{}}
```

**Comment by bhackett1024** [comment]
This works for me.

It looks to me like the controller finished both those requests:

```
{
  "controlId": "f8f4f537-98e6-47cf-987d-9bc37aa5b1e8",
  "kind": "control",
  "phase": "End",
  "messageId": 17,
  "sessionId": "f8f4f537-98e6-47cf-987d-9bc37aa5b1e8/ec9c8af0-9e2e-452c-88c9-3ee6381ae961",
  "pauseId": "121",
  "method": "DOM.getDocument",
  "level": "info",
  "message": "HandleMessage",
  "timestamp": "2021-01-22T14:41:35.561Z"
}
{
  "controlId": "f8f4f537-98e6-47cf-987d-9bc37aa5b1e8",
  "kind": "control",
  "phase": "End",
  "messageId": 23,
  "sessionId": "f8f4f537-98e6-47cf-987d-9bc37aa5b1e8/ec9c8af0-9e2e-452c-88c9-3ee6381ae961",
  "pauseId": "121",
  "method": "DOM.querySelector",
  "level": "info",
  "message": "HandleMessage",
  "timestamp": "2021-01-22T14:41:36.561Z"
}
```

Both the screenshot above and the controller log are showing errors relating to CSS.getAppliedRules, could those have broken the frontend?

```
{
  "controlId": "f8f4f537-98e6-47cf-987d-9bc37aa5b1e8",
  "name": "href",
  "spec": [
    {
      "name": "href",
      "description": "URL of the original style sheet.",
      "type": "string"
    },
    {
      "name": "startLine",
      "description": "Start line in the style sheet (1-indexed).",
      "type": "integer"
    },
    {
      "name": "startColumn",
      "description": "Start column in the style sheet (0-indexed).",
      "type": "integer"
    }
  ],
  "level": "info",
  "message": "TypeCheckMissingValue",
  "timestamp": "2021-01-22T14:23:36.967Z"
}
{
  "controlId": "f8f4f537-98e6-47cf-987d-9bc37aa5b1e8",
  "method": "CSS.getAppliedRules",
  "error": 11,
  "data": "href",
  "level": "error",
  "message": "TypeCheckResultFailed",
  "timestamp": "2021-01-22T14:23:36.967Z"
}
```

It looks like an `OriginalStyleSheetLocation` is missing an `href` property.  I think this was fixed in #1008.
